securitytxt-signer.sh

```diff
@@ -77,14 +81,25 @@ if ! [[ "$KEY" =~ ^0x[a-fA-F0-9]{8,40}$ ]]; then
 else
   KEY_INFO=$(gpg --list-secret-keys "$KEY" 2> >(sed $'s,.*,\e[33m&\e[m,'>&2))
 
-  KEY_EXPIRES=$(
-    echo "$KEY_INFO" \
-      | grep "sec" \
-      | grep -Eo 'expires:\ [0-9\-]+' \
-      | awk '{ print $2}' \
-      | date -Iseconds -u -f - \
-      | sed -e 's/+00:00$/Z/'
-    )
+  if (( IS_MAC )); then
+    KEY_EXPIRES=$(
+      echo "$KEY_INFO" \
+        | grep "sec" \
+        | grep -Eo 'expires:\ [0-9\-]+' \
+        | awk '{ print $2}' \
+        | date -Iseconds -u \
```

Comment:
Does this really take the time from the pipe and use that? Based on https://ss64.com/mac/date.html it would only print the **current** date. We don't want it to expire **now** but either after `$DAYS_MAX` or when the key expires; whichever comes sooner.

Reply:
"Providing arguments will format the date and time in a user-defined way"; will work, but I wouldn't detect MacOS but BSD version of Date.